x-pack/plugins/observability/public/components/threshold/components/expression_row.test.tsx

```diff
@@ -86,34 +87,15 @@ describe('ExpressionRow', () => {
       threshold: [0.5],
       timeSize: 1,
       timeUnit: 'm',
-      aggType: 'avg',
+      aggType: 'custom',
     };
     const { wrapper } = await setup(expression as MetricExpression);
     const [valueMatch] =
       wrapper
         .html()
-        .match('<span class="euiExpression__value css-1lfq7nz-euiExpression__value">0.5</span>') ??
-      [];
+        .match(
+          '<span class="euiExpression__value css-uocz3u-euiExpression__value-columns">0.5</span>'
+        ) ?? [];
     expect(valueMatch).toBeTruthy();
   });
-
-  it('should render a helpText for the of expression', async () => {
```

Comment:
This test is no longer relevant as we removed its related code, the OF expression